src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToDefenceCallbackHandler.java

```diff
@@ -62,6 +71,44 @@
         );
     }
 
+    private CallbackResponse populateRespondent1ClaimResponseDocumentCopy(CallbackParams callbackParams) {
+        var caseData = callbackParams.getCaseData();
+
+        CaseData.CaseDataBuilder updatedData =
+            caseData.toBuilder()
+                .respondent1ClaimResponseDocumentCopy(caseData.getRespondent1ClaimResponseDocument());
+
+        MultiPartyScenario multiPartyScenario = getMultiPartyScenario(caseData);
+
+        if (featureToggleService.isMultipartyEnabled()) {
```

Comment:
Don't need this featuretoggle anymore. There's another ticket enroute which removes this. So, we should not reintroduce this check

Reply:
Done